README.md

```diff
@@ -107,14 +107,14 @@ terraform apply -auto-approve -input=false -lock=true -parallelism=10 -refresh=t
 jobs:
   global:
     - name: Configure AWS credentials
-      uses: aws-actions/configure-aws-credentials@v1
+      uses: aws-actions/configure-aws-credentials@v4
       with:
         aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
         aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
         aws-region: eu-central-1
 
     - name: Setup Terraform
-      uses: hashicorp/setup-terraform@v1
+      uses: hashicorp/setup-terraform@v2
```

Comment:
I don't have a setup available to verify this, though I don't see any breaking changes listed: https://github.com/hashicorp/setup-terraform/releases/tag/v2.0.0